package.json

```diff
@@ -9,16 +9,25 @@
   },
   "scripts": {
     "start": "./entry.sh",
-    "build": "coffee -c src",
+    "build": "coffee -c src && webpack --optimize-minimize --define process.env.NODE_ENV=\"production\"",
```

Comment:
God, webpack is hard to configure. I had to add the `extensions` field to webpack.config.js to make it work.